scripts/testing/e2e-runner

```diff
@@ -0,0 +1,357 @@
+#!/bin/bash
+
+
+#
+# Default configuration, overridable on the command line.
+#
+# REMOTE_REPO:   remote repository to clone
+# CLONED_REPO:   local location to clone to
+# RESULT_ROOT:   directory for publishing results
+# RESULT_NAME:   subdirectory for publishing results of this test run
+# SKIP_WORKTREE: skip preparing worktree, assume it is already done
+# E2E_TESTS:     e2e tests to run
+
+REMOTE_REPO="${REMOTE_REPO:-https://github.com/containers/nri-plugins}"
+CLONED_REPO="${CLONED_REPO:-/opt/e2e-test/nri-plugins/nri-plugins}"
+TEST_BRANCH="${TEST_BRANCH:-main}"
+RESULT_ROOT="${RESULT_ROOT:-/opt/e2e-test/nri-plugins/results}"
+RESULT_NAME="${RESULT_NAME:-$(date +%Y-%m-%d-%H-%M)}"
```

Comment:
I'm wondering if it would be beneficial to have the latest commit's short SHA-1 added either here or somewhere else ?

Reply:
Yes definitely, and it is one of the things I now try to add by generating a more reasonable/useful summary/index for the results. It is visible in the logs, but I too would like to explicitly spell out the SHA1 in the generated summary.

Reply:
Updated the script to extract and store both report in the generated index the both the SHA1 and the result of `git describe` of the worktree, and to show these in the generated top-level index.